### PR TITLE
Test using Executable

### PR DESCRIPTION
**Pull request by ranjitjhala**
Tweak the test code to use the `liquid-rust` executable. Gets @nilehmann 's `heapsort.rs` test working.

### CODE REVIEW

File: tests/pos/rvec.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#![feature(register_tool)]
```
Comment (nilehmann) [inline tests/pos/rvec.rs:1]: Do we really need this here? I think we only need to register the tool in the crate root.

**Review by nilehmann — APPROVED** [review]

File: tests/pos/rvec.rs (hunk 1 above)
Comment (ranjitjhala) [reply tests/pos/rvec.rs]: Good point I removed it.

**Comment by ranjitjhala** [comment]
Any idea why the `neg` tests are failing in the CI ? It seems to complain about the paths, but AFAICT those files `tests/neg/test0{0,1}.rs` are already checked in? Can you try a `cargo test` locally?

**Comment by nilehmann** [comment]
@ranjitjhala is failing for me locally as well with the same error.

**Comment by ranjitjhala** [comment]
puzzling. let me make a fresh clone and try to replicate.

**Comment by nilehmann** [comment]
@ranjitjhala ok, it makes sense the `liquid-rust` binary won't be in `target/debug` unless you run `cargo build` (or `cargo run`). `cargo test` won't create the binary.

**Comment by ranjitjhala** [comment]
aha. So the issue is that you have to run a `cargo build` first and then
the `cargo test`.

(Odd that only the `neg` tests complain about it...)

**Comment by ranjitjhala** [comment]
@nilehmann -- any idea how to fix? Is there a way to tell the `cargo test` to make sure it does a `build` first? OR should we do it in the CI script?

**Comment by nilehmann** [comment]
@ranjitjhala changing the CI script to run `cargo build` first is the easiest.

**Pull request merged**